src/opensynth/models/faraday/new_gmm/train_gmm.py

```diff
@@ -58,6 +58,8 @@ def training_loop(
     converged = False
     lower_bound = -np.inf
 
+    log_prob_epochs = []
```

Comment:
I think we'll want to delete this function right when we're merging it into feature/gmm-lightning right? This is just a function to explicitly perform e/m step outside of torch lightning. If so - worth adding a #todo comment!

Reply:
Yep!